Content.Server/Chat/Systems/ChatSystem.Emote.cs

```diff
@@ -150,6 +152,10 @@ public bool TryPlayEmoteSound(EntityUid uid, EmoteSoundsPrototype? proto, string
     private void TryEmoteChatInput(EntityUid uid, string textInput)
     {
         var actionLower = textInput.ToLower();
+        // Replace ending punctuation with nothing
+        if (Punctuation.Any(punctuation => actionLower.EndsWith(punctuation)))
+            actionLower = actionLower.Remove(actionLower.Length - 1);
+
         if (!_wordEmoteDict.TryGetValue(actionLower, out var emote))
             return;
```

Comment:
I hope this isn't *actually* modifying the emotes. Because I would prefer that it doesn't, and that the system simply makes it not consider punctuation when deciding whether or not to play an emote sound.

Second, I'd like to request if possible, consider making it so that it reads the whole emote for a sound effect. I've been annoyed for a very long time that I can't include the word *PEW* in a full sentence emote, and the sound won't play.

Reply:
> I hope this isn't actually modifying the emotes. Because I would prefer that it doesn't, and that the system simply makes it not consider punctuation when deciding whether or not to play an emote sound.

Look at the video, it doesn't change the text.

> Second, I'd like to request if possible, consider making it so that it reads the whole emote for a sound effect. I've been annoyed for a very long time that I can't include the word PEW in a full sentence emote, and the sound won't play.

Do that in another PR.